tooling/noir_js_types/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "parser": "typescript",
+  "printWidth": 120,
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```

Comment:
This should be moved to cover the entire workspace for consistency. This creates a _lot_ of changes however so I'd recommend removing this file and handling this in a separate PR.

Reply:
Okay removing